migrations/versions/194a5a2a44ef_.py

```diff
@@ -0,0 +1,49 @@
+"""empty message
+
+Revision ID: 194a5a2a44ef
+Revises: 4cac94c86047
+Create Date: 2018-05-21 01:54:54.296053
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '194a5a2a44ef'
+down_revision = '4cac94c86047'
+
+def update_rating(rating):
+    """Updates the old rating to new rating."""
+    rating = float(rating)
+    if 0 <= rating <= 5:
+        return round(rating*2, 0) / 2  # Rounds to nearest 0.5
+    return 0
+
+
+def upgrade():
+    feedbacks_table = sa.sql.table('feedback',
+                                   sa.Column('rating', sa.VARCHAR())
+                                )
+    
+    op.execute(
+        feedbacks_table.update().values(
+            rating=str(update_rating(feedbacks_table.c.rating))
```

Comment:
This fails with the error
```
TypeError: <flask_script.commands.Command object at 0x103f520f0>: float() argument must be a string or a number, not 'Column'
```
How do I perform a self-correlated update here? What I wish to do is something like
```
UPDATE feedback SET rating = update_rating(rating);
```

Reply:
check this migration `versions/dbb9670cd902_.py`